docs/trace/links-creation-with-new-activities/README.md

```diff
@@ -0,0 +1,119 @@
+# Creating new root activities that link to an existing activity: An Example
+
+This example shows how to create new root activities that link to an existing
```

Comment:
it may not be obvious that "link to" here is referring to the OTel's concept of SpanLinks. Perhaps doc link helps?

Reply:
Good point, will add a sentence about it here, and will also add a reference to the span links spec.